src/test/java/com/salesforce/storm/spout/dynamic/VirtualSpoutTest.java

```diff
@@ -1507,54 +1460,6 @@ public void testCloseWithCompletedFlagSetToTrue() throws NoSuchFieldException, I
         verify(mockConsumer, never()).flushConsumerState();
     }
 
-    /**
-     * Test calling close, verifies what happens if the completed flag is true.
-     * Verifies what happens if SidelineRequestIdentifier is null.
-     */
-    @Test
-    public void testCloseWithCompletedFlagSetToTrueNoSidelineREquestIdentifier() throws NoSuchFieldException, IllegalAccessException {
```

Comment:
This test is a duplicate of the test directly above it.  I believe its a hold over from when we had sideline logic more closely intertwined into the VirtualSpout class.